app/src/main/java/app/foodfinderapp/dto/Restaurant.kt

```diff
@@ -1,8 +1,9 @@
 package app.foodfinderapp.dto
 
+import com.google.gson.annotations.SerializedName
 import java.util.Date
 
-data class Restaurant(val restaurantID : Int, var name:String,var category : String, var hours: String, var contact: String){
+data class Restaurant(val restaurantID : Int, @SerializedName("name") var name:String, @SerializedName("category") var category : String,@SerializedName("hours") var hours: String,@SerializedName("contact") var contact: String){
```

Comment:
See notes above.

@SerializedName is only required if the JSON field names do not match the data class attribute names.
In a case like this, where the names match, @SerializedName is not providing value... and thus, is extra code that is not needed.